pkg/cmd/pulumi/new.go

```diff
@@ -1037,6 +1038,7 @@ func promptForValue(
 func templatesToOptionArrayAndMap(templates []workspace.Template,
 	showAll bool) ([]string, map[string]workspace.Template) {
 
+	termWidth, _, _ := terminal.GetSize(0)
```

Comment:
If GetSize fails, [termWidth will be negative](https://cs.opensource.google/go/x/term/+/refs/tags/v0.4.0:term_unix.go;l=63;drc=1efcd90d861e239a7719db7012b81621e6f7d297).
Should we add a reasonable default (80?) if there's an error?

```suggestion
	termWidth, _, err := terminal.GetSize(0)
	if err != nil {
		termWidth = 80
	}
```

Reply:
Yup!